.pre-commit-config.yaml

```diff
@@ -15,6 +15,10 @@
 # See https://pre-commit.com for more information
 
 repos:
+  - repo: https://github.com/pre-commit/mirrors-clang-format
+    rev: v18.1.3
```

Comment:
This is the same version as Velox's one: https://github.com/facebookincubator/velox/blob/561d175a47f9eec55193d5d665f9e8b1972f1f50/.pre-commit-config.yaml#L76